EndpointCategory/helpers/instantiateEndpoint.js

```diff
@@ -70,84 +87,114 @@ module.exports = (config) => {
         ...endpointOptions
       } = options;
 
-      let response;
       // Initialize with generic error message
       let zoomErrorMessage = 'An unknown error occurred';
 
-      try {
+      // Declare pages array that will hold concatenated responses
+      const pages = [];
+
+      const fetchPage = async () => {
```

Comment:
Let's test the fetchPage feature during the next meeting. There might be odd behaviors that we're missing.